scripts/docker-helper/readme.md

```diff
@@ -149,24 +157,24 @@ Medic recommends setting up unique project names and unique ports for each proje
 
 ### Slow downloads and wait periods
 
-During testing on an Internet connection with high latency (>1000ms) and packet loss, this script had trouble booting the CHT instance because it was taking too long to download the assets from `staging.dev.medicmobile.org`.  Each version is about 38MB.  
+During testing on an Internet connection with high latency (>1000ms) and packet loss, this script had trouble booting the CHT instance because it was taking too long to download the assets from `staging.dev.medicmobile.org`.  Each version is about 38 MB.  
 
 To account for this, the wait time is multiplied times the boot iteration for each time it reboots.  It starts at 100 seconds and then 200, 300, 400 up to the fifth time it will wait 500 seconds. 
 
-## Too many containers
+### Too many containers
 
-If you're on a resource constrained computer, like a very old or very slow laptop, be sure to watch the total number of containers you're running.  More than one or two projects (2 or 4 containers) and you may notice a slow down. You can use the `./docker-status.sh` script if you forgot which projects you have running:
+If you're on a resource constrained computer, like a very old or very slow laptop, be sure to watch the total number of containers you're running.  More than one or two projects (2 or 4 containers) and you may notice a slow-down. You can use the `./docker-status.sh` script if you forgot which projects you have running:
```

Comment:
Unrelated to these changes, but if you have not seen it before, you might be interested in https://github.com/bcicen/ctop.  (Its goal is to provide similar insights into running containers)

Reply:
"[fixed](https://github.com/mrjones-plip/DockStat/blob/main/readme.md#related-work)" ;)